docs/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -391,6 +391,22 @@ SentrySDK.reportFullyDisplayed()
 If the span finishes through the API, its `status` is set to `SpanStatus.OK`. If the span doesn't finish after 30 seconds, it will be finished by the SDK automatically, and its `status` will be set to `SpanStatus.DEADLINE_EXCEEDED`, also its duration will match the same of the `Time to initial display` span and the description will contain `Deadline Exceeded` suffix.
 If a call to `reportFullyDisplayed()` happens before the view controller appears, the reported time will be shifted to `Time to initial display` measured time.
 
+## Performance V2
+
+<Note>
+
+This feature is experimental and may have bugs.
+
+</Note>
+
+_(New in version 8.18.0)_
+
+We are improving the performance offering with the code name Mobile Starfish, which aims to provide better insights into mobile app performance and highlight clear actions to improve app performance for developers. The `performanceV2` option changes the following behavior:
+
+* The app start duration finishes when the first frame is drawn instead of when the OS posts the `UIWindowDidBecomeVisibleNotification`.
+
+We aim to include the behavioral changes by default in the next major version.
```

Comment:
```suggestion
We're working to update our **Performance** product offering in order to be able to provide better insights and highlight specific actions you can take to improve your mobile app's overall performance. The `performanceV2` option changes the following behavior:

* The app start duration will now finish when the first frame is drawn instead of when the OS posts the `UIWindowDidBecomeVisibleNotification`.

This change will be the default in the next major version.
```